### ISSUE TITLE
Cannot find type 'RCTBridge' in scope

### ISSUE THREAD

**Issue by ronaldlangeveld** [MaxAst/expo-share-extension#61]
Hi!

Currently learning React Native for fun over weekends and a few months ago I tried `expo-share-extension` it worked perfectly fine.

However the past few weeks I've been coming across this as soon as I install the extension.

This morning again, on a brand now project, with pretty much nothing else installed and then running `npx expo run:ios`.

```
...ViewController.swift:16:36)

  14 | class ShareExtensionViewController: UIViewController {
  15 |   
> 16 |   private static var sharedBridge: RCTBridge?
     |                                    ^ cannot find type 'RCTBridge' in scope
  17 |   private weak var rootView: RCTRootView?
  18 |   private let loadingIndicator = UIActivityIndicatorView(style: .large)
  19 |   
  
  CommandError: Failed to build iOS project. "xcodebuild" exited with error code 65.
```

I'm experimenting with a standard Expo development build (not Expo Go) and then installing `expo-share-extension` following the instructions on your readme and then running `npx expo prebuild` followed by `npx expo run:ios`, which then outputs the error above.

Not sure if it's a me problem, given my limited React Native knowledge or if I'm doing something completely wrong or if we're missing something in the docs?

Running MacOS, latest everything (including Expo & extension version).
I disabled new arch for in case as well.

Thanks in advance 🙏🏻

**Comment by MaheshtheDev** [comment]
@MaxAst , thanks. Having same issue! should we downgrade or are you working on any quick fix?

**Comment by ronaldlangeveld** [comment]
Thank you very much getting back to us @MaxAst! 🙏

**Comment by MaxAst** [comment]
QQ, did you disable new arch in app.json?

![Screenshot 2024-12-10 at 09 34 26](https://github.com/user-attachments/assets/7f3742fb-ba06-448c-bc07-d300b9c9f430)

**Comment by MaheshtheDev** [comment]
I already disabled it @MaxAst

**Comment by MaxAst** [comment]
okay, working on a fix, but not sure how long it'll take. If possible, downgrading to expo v51 would definitely be the quickest unblock for you, if possible

**Comment by MaxAst** [comment]
@MaheshtheDev @ronaldlangeveld this is now fixed in 2.0.1, please upgrade! Apologies for the hiccup

**Comment by MaheshtheDev** [comment]
Thanks for quick response and fix @MaxAst
